javav2/usecases/creating_fsa_app/README.md

```diff
@@ -0,0 +1,989 @@
+
+        Region region = Region.US_EAST_1;
+        LambdaClient awsLambda = LambdaClient.builder()
+            .region(region)
+            .build();
+
+        // Update all four Lambda functions.
+        updateFunctionCode(awsLambda, analyzeSentimentLambda, bucketName, key);
+        updateFunctionCode(awsLambda, synthesizeAudioLambda, bucketName, key);
+        updateFunctionCode(awsLambda, extractTextLambda, bucketName, key);
+        updateFunctionCode(awsLambda, translateTextLambda, bucketName, key);
+        System.out.println("You have successfully updated the AWS Lambda functions");
+    }
+
+    public static void updateFunctionCode(LambdaClient awsLambda, String functionName, String bucketName, String key) {
+        try {
+            LambdaWaiter waiter = awsLambda.waiter();
+            UpdateFunctionCodeRequest functionCodeRequest = UpdateFunctionCodeRequest.builder()
+                .functionName(functionName)
+                .publish(true)
+                .s3Bucket(bucketName)
+                .s3Key(key)
+                .build();
+
+            UpdateFunctionCodeResponse response = awsLambda.updateFunctionCode(functionCodeRequest) ;
+            GetFunctionConfigurationRequest getFunctionConfigRequest = GetFunctionConfigurationRequest.builder()
+                .functionName(functionName)
+                .build();
+
+            WaiterResponse<GetFunctionConfigurationResponse> waiterResponse = waiter.waitUntilFunctionUpdated(getFunctionConfigRequest);
+            waiterResponse.matched().response().ifPresent(System.out::println);
+            System.out.println("The last modified value is " +response.lastModified());
+
+        } catch(LambdaException e) {
+            System.err.println(e.getMessage());
+            System.exit(1);
+        }
+    }
+}
+```
+
+## Deploy the AWS resources
+
+At this point, you have completed all of the application Java business logic required for the FSA application to work. Now you need to deploy the AWS resources, including the AWS Lambda functions and API Gateway endpoints. 
+
+Instead of deploying all of the resources manually by using the AWS Management Console, you can use a provided AWS CDK script. The script makes it more efficient to deploy the resources. 
+
+**Note**: For information about the AWS CDK, see [What is the AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/home.html).
+		
+For complete instuctions on how to run the supplied AWS CDK script, see [Feedback Sentiment Analyzer (FSA)](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/applications/feedback_sentiment_analyzer/README.md).
+
+### Update the Lambda functions
+
+After you execute the AWS CDK script, the Lambda functions are created. However, you can replace the ones installed by using the AWS CDK with newer versions if you desire. For example, if you want to add more logging functionality, you can modify the Java Handler classes and run **ModifyLambda** without having to run the CDK script again. 
```

Comment:
I understand you've put this in a section labeled "optional", but we've discussed the purpose and intent of these sample apps is to show a holistic project. That's not to say there isn't a place for your ModifyLambda tool, but would it be better to leave that in the use_cases for Lambda and leave this example focused entirely on the spec & using the CDK?

Reply:
I will take out this.